app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -104,6 +104,6 @@
   <section class="col col-space-m col-720">
```

Comment:
@emmaachesongray Bit confused why this file is still in ITRFM - shouldn't it have been moved to LAAT rather than this file still existing and a new duplicate file having been created?